### PR TITLE
added imagenette_classes

### PR DESCRIPTION
**Pull request by always-newbie161**
Added imagenette_names as a CSV file

### CODE REVIEW

**Review by mjsML — CHANGES_REQUESTED** [review]
I'm wondering why we should have those? Isn't this part of the TFDS data?

File: data/imagenette_class_names.csv [hunk 1]
```diff
@@ -0,0 +1,11 @@
+label,name
+0,Tench Tinca
+1,English springler
```
Comment (murphyk) [inline data/imagenette_class_names.csv]: typo: springer

File: data/imagenette_class_names.csv [hunk 2]
```diff
@@ -0,0 +1,11 @@
+label,name
+0,Tench Tinca
+1,English springler
+2,casette_player
```
Comment (murphyk) [inline data/imagenette_class_names.csv]: typo: cassette

File: data/imagenette_class_names.csv [hunk 3]
```diff
@@ -0,0 +1,11 @@
+label,name
```
Comment (murphyk) [inline data/imagenette_class_names.csv:1]: Please be consistent with capitalization, and use of underscores to join words.

**Review by murphyk — CHANGES_REQUESTED** [review]

**Review by murphyk — CHANGES_REQUESTED** [review]
you are still inconsistent with capitalization.
I suggest lower case for everything.

**Pull request merged**